misc/config_tools/schema/types.xsd

```diff
@@ -11,6 +13,13 @@
   </xs:restriction>
 </xs:simpleType>
 
+<xs:simpleType name="EnablementType">
```

Comment:
This type is defined but not used.  Will you be replacing some of the Boolean options with this type?

Reply:
It is used as the type of vuart0 and vbootloader.

The reason I introduce this is the same as why I add texts to the log levels. Perhaps we can keep using Boolean here, and label the two Boolean values as "Enable" and "Disable", if they make sense in all config items of that type.